### PR TITLE
Add a concurrency example

### LINKED ISSUE #6: Example on Concurrency

### PR DESCRIPTION
**Pull request by itmecho**
Fixes #6

### CODE REVIEW

File: src/concurrency.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	"net/http"
+)
+
+type response struct {
+	Code int
+	URL  string
+	Err  error
+}
+
+func main() {
```
Comment (sagar-jadhav) [inline src/concurrency.go:14]: Add description here about program. For e.g. 
- Purpose of program
- How concurrency achieve here?
- Call out packages & special methods used in the program

**Review by sagar-jadhav — CHANGES_REQUESTED** [review]

**Comment by sagar-jadhav** [comment]
Thank you @itmecho for contribution 👍

**Comment by sagar-jadhav** [comment]
Assign #6 to yourself

File: src/concurrency.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+			Err: err,
+		}
+		return
+	}
+
+	responses <- response{
+		URL:  url,
+		Code: resp.StatusCode,
+	}
+}
```
Comment (rajkumarGosavi) [inline src/concurrency.go:52]: One can try dividing this example in 2 parts.
1. Only explaining about Go routines
2. Go routines + channels

**Review by rajkumarGosavi — CHANGES_REQUESTED** [review]
Can elaborate more to make the example easy to understand for first time developers new to the topic of concurrency

**Comment by sagar-jadhav** [comment]
Closing the #14 as issue got already resolved via #82.

Thank you @itmecho for your contribution 👍

**Pull request closed without merging**